bam_masterdata/datamodel/object_types.py

```diff
@@ -8722,6 +8722,46 @@ class Weldment(ExperimentalStep):
         section="Identifiers",
     )
 
+    weld_joint_number = PropertyTypeAssignment(
+        code="EXPERIMENTAL_STEP.WELD_JOINT_NUMBER",
```

Comment:
Any reason why the code starts with EXPERIMENTAL_STEP?

Reply:
update codes to remove `EXPERIMENTAL_STEP`